bugsnag-plugin-android-ndk/src/androidTest/java/com/bugsnag/android/ndk/migrations/EventMigrationTest.kt

```diff
@@ -0,0 +1,39 @@
+package com.bugsnag.android.ndk.migrations
+
+import android.content.Context
+import androidx.test.platform.app.InstrumentationRegistry
+import com.fasterxml.jackson.databind.ObjectMapper
+import org.junit.Before
+import java.io.File
+
+open class EventMigrationTest {
+
+    private lateinit var context: Context
+    private var objectMapper = ObjectMapper()
```

Comment:
This doesn't appear to be re-assigned anywhere, so maybe it can be a `val`?
```suggestion
    private val objectMapper = ObjectMapper()
```